### ISSUE TITLE
Error when creating filter for storage filename

### ISSUE THREAD

**Issue by nazikus** [HumanSignal/label-studio#3337]
**Describe the bug**
When creating a new view filter based on storage filename, exception pops up.

**To Reproduce**
1. Go to Project view
2. Click on 'Filters', select "Storage filename" "contains"
3. Exception pop ups

**Expected behavior**
Expect to filter tasks by storage filename filter.

**Screenshots**
![image](https://user-images.githubusercontent.com/2558905/204758817-e59f82bd-73af-4692-a5ba-9babe303da3d.png)

**Environment (please complete the following information):**
 - OS: Ubuntu 22.04, docker image tag 1.6.0 
 - Label Studio Version 1.6.0rc5

**Additional context**

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1955, in add_fields
    join_info = self.setup_joins(name.split(LOOKUP_SEP), opts, alias, allow_many=allow_m2m)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1648, in setup_joins
    path, final_field, targets, rest = self.names_to_path(
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1562, in names_to_path
    raise FieldError("Cannot resolve keyword '%s' into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'storage_filename' into field. Choices are: annotations, cancelled_annotations, comment_authors, comment_count, created_at, data, drafts, file_upload, file_upload_id, id, inner_id, io_storages_azureblobimportstoragelink, io_storages_gcsimportstoragelink, io_storages_localfilesimportstoragelink, io_storages_redisimportstoragelink, io_storages_s3importstoragelink, is_labeled, last_comment_updated_at, locks, meta, overlap, predictions, project, project_id, total_annotations, total_predictions, unresolved_comment_count, updated_at, updated_by, updated_by_id

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/django/utils/decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "/label-studio/label_studio/data_manager/api.py", line 202, in get
    queryset = self.get_task_queryset(request, prepare_params)
  File "/label-studio/label_studio/data_manager/api.py", line 173, in get_task_queryset
    return Task.prepared.only_filtered(prepare_params=prepare_params)
  File "/label-studio/label_studio/data_manager/managers.py", line 630, in only_filtered
    return queryset.prepared(prepare_params=prepare_params)
  File "/label-studio/label_studio/data_manager/managers.py", line 451, in prepared
    queryset = apply_filters(queryset, prepare_params.filters, project, request)
  File "/label-studio/label_studio/data_manager/managers.py", line 326, in apply_filters
    value_type = type(queryset.values_list(field_name, flat=True)[0]).__name__
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/query.py", line 867, in values_list
    clone = self._values(*_fields, **expressions)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/query.py", line 835, in _values
    clone.query.set_values(fields)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 2272, in set_values
    self.add_fields(field_names, True)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1982, in add_fields
    raise FieldError("Cannot resolve keyword %r into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'storage_filename' into field. Choices are: annotations, cancelled_annotations, comment_authors, comment_count, created_at, data, drafts, file_upload, file_upload_id, id, inner_id, io_storages_azureblobimportstoragelink, io_storages_gcsimportstoragelink, io_storages_localfilesimportstoragelink, io_storages_redisimportstoragelink, io_storages_s3importstoragelink, is_labeled, last_comment_updated_at, locks, meta, overlap, predictions, project, project_id, total_annotations, total_predictions, unresolved_comment_count, updated_at, updated_by, updated_by_id
```

**Comment by makseq** [comment]
@nazikus Source filename is a column for displaying only, filters are not supported. If you could contribute, it would be great :)

**Comment by makseq** [comment]
Related: https://github.com/heartexlabs/label-studio/issues/2062

**Comment by kemics** [comment]
this is duplicate of https://github.com/heartexlabs/label-studio/issues/3076

**Comment by jombooth** [comment]
Hi @nazikus - this was fixed in our latest release, [1.11.0](https://github.com/HumanSignal/label-studio/releases/tag/1.11.0)!
